src/interfaces/IValidation.sol

```diff
@@ -6,26 +6,28 @@ import {PackedUserOperation} from "@eth-infinitism/account-abstraction/interface
 import {IPlugin} from "./IPlugin.sol";
 
 interface IValidation is IPlugin {
-    /// @notice Run the user operation validationFunction specified by the `functionId`.
-    /// @param functionId An identifier that routes the call to different internal implementations, should there be
+    /// @notice Run the user operation validationFunction specified by the `entityId`.
+    /// @param entityId An identifier that routes the call to different internal implementations, should there
+    /// be
     /// more than one.
```

Comment:
Nit: inline comment here, and in the other 2 places in this file with the same line split of `/// be`